sdk/core/Azure.Core/src/NullableResponseOfT.cs

```diff
@@ -40,6 +81,79 @@ public abstract class NullableResponse<T>
         public override int GetHashCode() => base.GetHashCode();
 
         /// <inheritdoc />
-        public override string ToString() => $"Status: {GetRawResponse()?.Status}, Value: {(HasValue ? Value : NoValue)}";
+        public override string ToString()
+             => $"Status: {GetRawResponse()?.Status}, Value: {(HasValue ? Value : NoValue)}";
+
+        private static Response ReplaceWithDefaultIfNull(Response? response)
```

Comment:
Why do we have that? i.e. why do we need to enable creating this type with null Responses? We should always have a response, shouldn't we?

Reply:
`NullableResponse<T>` has a GA protected parameterless constructor: https://learn.microsoft.com/en-us/dotnet/api/azure.nullableresponse-1.-ctor?view=azure-dotnet

Reply:
Ah, the wonders of implicit ctors :-(

Reply:
I know, sadness!